src/rp2_common/pico_standard_link/CMakeLists.txt

```diff
@@ -94,4 +94,8 @@ if (NOT TARGET pico_standard_link)
         target_compile_options(pico_standard_link INTERFACE -ffunction-sections -fdata-sections)
         target_link_options(pico_standard_link INTERFACE "LINKER:--gc-sections")
     endif()
+
+    if (CMAKE_C_COMPILER_VERSION VERSION_GREATER 12)
```

Comment:
This isn't a great check. A) the warning actually comes from binutils 2.39 rather than any particular C compiler version, and B) clang also exists with version numbers greater than 12.

Unfortunately cmake doesn't give a good way to get the linker version so a proper check here probably needs some sort of way to parse the `ld --version` output.

Reply:
Thanks. I'll take another look.